### PR TITLE
Fix displaying products of the new product page when no product is new

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | After setting the "number of days a product is considered as New Product" to "0" the new product page always displaying all products as new. Problem solved by setting the nb_days_new_product to 0 when PS_NB_DAYS_NEW_PRODUCT is set to 0. |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1043 |
| How to test? | Just set the "number of days a product is considered as New Product" to "0" and refresh the new products page. |

### CODE REVIEW

**Comment by kpodemski** [comment]
i don't think that it is needed, maybe we should just return empty array if someone don't want to display new products?

File: classes/Product.php [hunk 1]
```diff
@@ -2250,6 +2250,7 @@ public function hasAttributes()
     */
     public static function getNewProducts($id_lang, $page_number = 0, $nb_products = 10, $count = false, $order_by = null, $order_way = null, Context $context = null)
     {
+        $now = date('Y-m-d') . ' 00:00:00';
```
Comment (mickaelandrieu) [inline classes/Product.php]: Why not use ``date('Y-m-d H:i:s')`` ?

Comment (julienbourdeau) [inline classes/Product.php]: Because the option works on a day basis, so I guess it's enough.

**Comment by Shudrum** [comment]
Thank you!

**Pull request merged**